doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -44,354 +44,1448 @@ This resource allows you to obtain a list of subresources published by the API:
             }
-         }
+          }
         ]
+  
 
 ### By pattern matching [GET /v2/entities{?idPattern,options}]
 
 + Parameters
-    + idPattern: DC_.* (optional, string) - Only retrieve entities having a matching id.
+    + idPattern: 0115206c51f60b48b77e4* (optional, string) - Only retrieve entities having a matching id.
     + options: keyValues (optional, string) - rendering options
 
++ Request 
+    
+    + Headers
+
+            Fiware-Service: tourguide
+
 + Response 200 (application/json)
 
         [
-         {
-            "type": "Room",
-            "id": "DC_S1_D41",
-            "temperature": 35.6
-         }
+          {
+            "id": "0115206c51f60b48b77e4c937835795c33bb953f",
+            "type": "Restaurant",
+            "address":
+            {
+              "streetAddress": "Cuesta de las Cabras Aldapa 2",
+              "addressRegion": "Araba",
+              "addressLocality": "Alegría-Dulantzi",
+              "postalCode": "01240"
+            },
+            "aggregateRating":
+            {
+              "reviewCount": 1,
+              "ratingValue": 3
+            },
+            "capacity": 120,
+            "department": "Franchise2",
+            "description": "Restaurante de estilo sidrería ubicado en Alegria-Dulantzi. Además ...",
+            "location": "42.8404625, -2.5123277",
+            "name": "Elizalde",
+            "occupancyLevels": 0,
+            "priceRange": 0,
+            "telephone": "945 400 868"
+          }
         ]
 
+
+
 # Group Context Entity
 
 ## Retrieve entity data [/v2/entities/{entityId}]
 
 ### By Id [GET /v2/entities/{entityId}{?options}]
 
 + Parameters
-    + entityId: Bcn_Welt (required, string) - Entity Id.
+    + entityId: 0115206c51f60b48b77e4c937835795c33bb953f (required, string) - Entity Id.
     + options: keyValues (optional, string) - rendering options
 
++ Request 
+    
+    + Headers
+
+            Fiware-Service: tourguide
+
 + Response 200 (application/json)
 
         {
-            "type": "Room",
-            "id": "Bcn_Welt",
-            "temperature": 21.7,
-            "humidity": 60
+          "id": "0115206c51f60b48b77e4c937835795c33bb953f",
+          "type": "Restaurant",
+          "address":
+          {
+            "streetAddress": "Cuesta de las Cabras Aldapa 2",
+            "addressRegion": "Araba",
+            "addressLocality": "Alegría-Dulantzi",
+            "postalCode": "01240"
+          },
+          "aggregateRating":
+          {
+            "reviewCount": 1,
+            "ratingValue": 3
+          },
+          "capacity": 120,
+          "department": "Franchise2",
+          "description": "Restaurante de estilo sidrería ubicado en Alegria-Dulantzi. Además...",
+          "location": "42.8404625, -2.5123277",
+          "name": "Elizalde",
+          "occupancyLevels": 0,
+          "priceRange": 0,
+          "telephone": "945 400 868"
         }
 
+
+
 ### By Id. Not exists [GET /v2/entities/{entityId}]
```

Comment:
Maybe we should write 'Non existing entity' or 'Non existing' instead of 'Not exists'.